tests/unit/processor/field_manager/test_field_manager.py

```diff
@@ -187,7 +187,7 @@
             "field3": ["value5", "value6"],
             "new_field": ["i exist"],
         },
-        {"new_field": ["i exist", "value1", "value2", "value3", "value4", "value5", "value6"]},
+        {"new_field": ["i exist", "value1", "value2", "value3", "value5", "value6", "value4"]},
```

Comment:
the fields should be appended as the ordering in source_fields.

```suggestion
        {"new_field": ["i exist", "value1", "value2", "value3", "value4", "value5", "value6"]},
```